CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing to Sharezone
+_Note: The `CONTRIBUTING.md` is in the process of making. See [#28](https://github.com/SharezoneApp/sharezone-app/issues/28) for the missing sections._
+
+## Welcome
+We love, that you are interested in contributing to Sharezone 💙 There are many ways to contribute and we appreciate all of them. This document gives a rough overview of how you can contribute to Sharezone and which steps you need to follow to set up the development environment.
+
+* [How to set up your development environment](#how-to-set-up-your-development-environment)
+
+If you have ans questions, please join our [Discord](https://sharezone.net/discord).
+
+## How to set up your development environment
+If you would like to contribute to our code, please follow the introduction to set up your development environment.
+
+### Operating system
+_tl:dr: We recommend macOS._
+
+You can use the operating system you like. But we recommend to use macOS because you might have some issues with other operating systems.
+
+**Known issues:**
+* Golden tests are only passing with macOS
+* Sharezone CLI has official support for macOS and Windows (should also work with other operating systems, but might cause problems in some cases)
```

Comment:
```suggestion
* The Sharezone CLI (used for development) only officially supports macOS and Windows (should also work with other operating systems, but might cause problems in some cases)
```

Just so it makes more sense as a "known issue". Also added the "(used for development)" since the general explanation for the CLI is located further below and hasn't been read yet.